src/services/lifi/api.ts

```diff
@@ -271,8 +272,9 @@ export class LiFiAPI {
     }
 
     if (response.status === 429) {
-      const error = `Our service provider received too many requests, temporarily preventing your request from being processed. ${errorPrefix}`
-      throw new SwapAndBridgeProviderApiError(error)
+      const error =
+        'Our service provider received too many requests, temporarily preventing your request from being processed.'
+      throw new SwapAndBridgeProviderApiError(error, 'Too many requests')
```

Comment:
The short note is too generic. You want to keep it concise so that in the UI it is on 1 line, right?

If so, I'd suggest: "Rate limit reached, try again in a while" maybe?